schism/compiler.ss

```diff
@@ -66,7 +66,7 @@
             (set-diff (cdr set) sub)
             (cons (car set) (set-diff (cdr set) sub)))))
 
-  (define (runtime-imports)
+  (define runtime-imports
```

Comment:
This is cool to be able to do this as a variable now instead of having to call a function!